customer-service/src/main/scala/gateway/serialization/CustomerCodecs.scala

```diff
@@ -1,7 +1,7 @@
 package gateway.serialization
 
 import domain.{Customer, CustomerId, CustomerName}
-import gateway.models.CustomerApi.{CustomerApiInput, CustomerApiOutput}
+import gateway.models.GatewayModels.{CustomerApiInput, CustomerApiOutput}
 import io.circe.Codec
 import io.circe.generic.semiauto.deriveCodec
 import io.scalaland.chimney.Transformer
```

Comment:
don't keep codecs separated, there is no need for that if u have pure "output api" objects, so instead of:
```
object CustomerCodecs {
  implicit val customerApiOutput: Codec[CustomerApiOutput] = deriveCodec
  implicit val customerApiInput: Codec[CustomerApiInput]   = deriveCodec
}
```

it should be side by side with object itself:
```
final case class CustomerApiOutput(id: String, name: String)
object CustomerApiOutput {
   // here codec
}
```